### PR TITLE
Add Yandex icon for browser list

### PR DESCRIPTION
**Pull request by spylogsster**
Resolves https://github.com/brave/brave-browser/issues/27760

image

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

- Steps from issue

### CODE REVIEW

File: components/brave_welcome_ui/components/svg/browser-icons/yandex.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+const SvgComponent = (props: any) => (
+<svg viewBox="0 0 200 200" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (chromium-santa) [inline components/brave_welcome_ui/components/svg/browser-icons/yandex.tsx:9]: @aguscruiz can you pls help to get correct svg? This seems a bit bigger than expected, thanks

Comment (spylogsster) [inline components/brave_welcome_ui/components/svg/browser-icons/yandex.tsx:9]: @aguscruiz pls help with correct svg, this seems a bit bigger than expected, thanks

Comment (spylogsster) [reply components/brave_welcome_ui/components/svg/browser-icons/yandex.tsx]: Thanks!

**Review by nullhook — APPROVED** [review]
lgtm

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.49.37 Chromium: 109.0.5414.87 (Official Build) nightly (x86_64)
-- | --
Revision | 2dc18eb511c56e012081b4abc9e38c81c885f7d4-refs/branch-heads/5414@{#1241}
OS | macOS Version 11.7.2 (Build 20G1020)

### Steps:

1. populated a Yandex browser with at least bookmarks
2. installed `1.49.37`
3. launched Brave
4. on `brave://welcome` looked for Yandex as an import option

#### Confirmed the `Yandex` logo and name show up in `brave://welcome` and profiles are imported successfully

`brave://welcome` | `Yandex profiles` | `all profiles selected` | `Setup complete` | `brave://extensions` | `brave://history` | `brave://bookmarks`
---------|----------|---------|----------|---------|----------|---------
Screen Shot 2023-01-17 at 1 45 27 PM | Screen Shot 2023-01-17 at 1 55 47 PM | Screen Shot 2023-01-17 at 1 55 56 PM | Screen Shot 2023-01-17 at 1 56 00 PM | Screen Shot 2023-01-17 at 1 56 14 PM | Screen Shot 2023-01-17 at 1 56 20 PM | Screen Shot 2023-01-17 at 1 56 43 PM

**Pull request merged**